manifests/m/Microsoft/msodbcsql/18.3.3.1/Microsoft.msodbcsql.installer.yaml

```diff
@@ -0,0 +1,29 @@
+PackageIdentifier: Microsoft.msodbcsql
+PackageVersion: 18.3.3.1
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://download.microsoft.com/download/4/f/e/4fed6f4b-dc42-4255-b4b4-70f8e2a35a63/en-US/18.3.3.1/x64/msodbcsql.msi
+    InstallerSha256: 680ef2c14a6b140ac292cf65a7a5b929035dda742095172360463e45e6e5acf4
+    InstallerType: msi
+    InstallerLocale: en-US
+    InstallerSwitches:
+      Silent: /quiet IACCEPTMSODBCSQLLICENSETERMS=YES
+      SilentWithProgress: /passive IACCEPTMSODBCSQLLICENSETERMS=YES
+  - Architecture: x86
+    InstallerUrl: https://download.microsoft.com/download/4/f/e/4fed6f4b-dc42-4255-b4b4-70f8e2a35a63/en-US/18.3.3.1/x86/msodbcsql.msi
+    InstallerSha256: 6f44c3d46f8ea5a6ab86cdcabe631233d7134c602e2a20ffb81c77d51ab6f6dc
+    InstallerType: msi
+    InstallerLocale: en-US
+    InstallerSwitches:
+      Silent: /quiet IACCEPTMSODBCSQLLICENSETERMS=YES
+      SilentWithProgress: /passive IACCEPTMSODBCSQLLICENSETERMS=YES
+  - Architecture: arm64
+    InstallerUrl: https://download.microsoft.com/download/4/f/e/4fed6f4b-dc42-4255-b4b4-70f8e2a35a63/en-US/18.3.3.1/arm64/msodbcsql.msi
+    InstallerSha256: 769eb91172b997c0b943c8261a86509b81ebba1fd5c1619955d2a874ce85f9ea
+    InstallerType: msi
+    InstallerLocale: en-US
+    InstallerSwitches:
+      Silent: /quiet IACCEPTMSODBCSQLLICENSETERMS=YES
```

Comment:
Because this is an agreement being passed without user knowledge, you will need to include the `Agreements` field in the manifest's Locale files